bazel/README.md

```diff
@@ -916,7 +916,27 @@ Edit the paths shown here to reflect the installation locations on your system:
 export CLANG_FORMAT="$HOME/ext/clang+llvm-12.0.1-x86_64-linux-gnu-ubuntu-16.04/bin/clang-format"
 export BUILDIFIER_BIN="/usr/bin/buildifier"
 ```
-
+The easiest way to use the correct `clang-format` in your host system is to copy the `clang-format` from the ci docker image.
+* Run the ci docker image
+```shell
+ci/run_envoy_docker.sh bash
+```
+* Get the docker container ID
+```shell
+dockerContainerID=$(docker ps | grep envoy-build-ubuntu | awk '{print $1}')
+```
+* Copy the `clang-format` to host machine
+```shell
+docker  copy $dockerContainerID:/opt/llvm/bin/clang-format clang-format-ci
+```
+* Replace the host `clang-format` with the new one. Ensure that the copied `clang-format` is the default one. You can do this by ensuring it is in `$PATH` or by creating a symbolic link:
+```shell
+copy clang-format-ci $PATH/clang-format
```

Comment:
`$PATH` cant be used as a path as it has multiple paths

i think the cannonical way to do this is probably to copy the bin to `/usr/local/bin` - ie

```console
cp clang-format-ci /usr/local/bin/clang-format
```

or as a (non-root) alternative using a bin dir and adding that to `$PATH`

```
mkdir bin
mv clang-format-ci bin/
PATH=$PATH:$PWD/bin/
```